src/framework/audio/worker/internal/startworkercontroller.h

```diff
@@ -54,6 +54,8 @@ class StartWorkerController : public IStartWorkerController
     void init(const OutputSpec& outputSpec, const AudioWorkerConfig& conf) override;
     void deinit() override;
 
+    void process(float* stream, unsigned samplesPerChannel);
```

Comment:
Perhaps we should rename this class, because now it does more than just start the worker

Reply:
I agree, I'll think about it next time.